jobs/service-fabrik-postgresql-manager/templates/bin/service-fabrik-postgresql-manager_ctl.erb

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e # exit immediately if a simple command exits with a non-zero status
+
+source /var/vcap/packages/bosh-helpers/ctl_setup.sh 'service-fabrik-postgresql-manager'
```

Comment:
i would rather have the job name as `service-fabrik-postgresql-multitenant-operator`

We would like to change our jobs also as operators. Now that you are adding one, please make it operator.

Reply:
@jagadish-kb makes sense.

Reply:
A very naive question: why does this change need to go into the open-source repo at all? how does this help someone who wishes to deploy fabrik from the github repository?

Reply:
Considering that the agent implementation guideline SF provides, if someone adheres to that in their agent implementation, then they can use the MT operator. Right?